### DISCUSSION TITLE
graphql/scalars/docs/scalars/date

### DISCUSSION THREAD

**Discussion by giscus** [Urigo/graphql-scalars#2125] [Docs Discussions]
# graphql/scalars/docs/scalars/date

GraphQL Scalars Documentation

https://the-guild.dev/graphql/scalars/docs/scalars/date

**Comment by ankit9787** [comment]
Hey do you have any code or sample code repo which we can run and see how custom scalar works ?? because theortically hard to understand its usage

**Reply by ardatan** [reply]
https://the-guild.dev/graphql/tools/docs/scalars
